mslib/_tests/test_utils.py

```diff
@@ -121,7 +121,7 @@ def test_existing_empty_config_file(self):
         with fs.open_fs(MSS_CONFIG_PATH) as file_dir:
             file_content = file_dir.readtext("mss_settings.json")
         assert ":" not in file_content
-        config_file = fs.path.join(MSS_CONFIG_PATH, "mss_settings.json")
+        config_file = fs.path.combine(MSS_CONFIG_PATH, "mss_settings.json")
```

Comment:
I understand that this is more efficient?

Reply:
this keeps the double slash, while the other removes one.

Reply:
There is nothing in the documentation for that.

This method only supports fs-paths.

fs can only deal with PyFilesystem paths.
Conversion works using

unsyspath and getsyspath methods.

https://pyfilesystem.readthedocs.io/en/latest/osfs.html

This should solve our "problem" with QT. But for the JSON-config we need some speculative conversion routine, because we may have either an OS or a fs-path.

I am getting doubts if this is even fine for a hotfix.